packages/components/NavBar.tsx

```diff
@@ -35,7 +35,7 @@ const actionButtons = [
 function NavBar() {
   const { t } = useI18n("navbar");
   const { isOpen, onOpen, onClose } = useDisclosure();
-  const navbarBgColor = useColorModeValue("gray.50", "gray.900");
+  const navbarBgColor = useColorModeValue("bg-gray-50", "bg-gray-900");
```

Comment:
**question**: `useColorModeValue` is a dependency from Chakra, does it work as expected for `tailwind`?

Reply:
Yes, in this context, `useColorModeValue` is being used to choose between the color schemes that will set the value of `navbarBgColor`.

When the `ToggleThemeButton` component is migrated, we'll need to revisit the `NavBar` component to change how we "read the color scheme" selected by the user.

Reply:
Sounds good, just keep in mind that we'll need to remove "useColorModeValue" and manage the dark theme as Tailwind does.

- https://tailwindcss.com/docs/dark-mode
- https://tailwindcss.com/docs/dark-mode#toggling-dark-mode-manually

Reply:
Yes, this change will be made when removing the Chakra.